lib/marked.js

```diff
@@ -7,8 +7,223 @@
 ;(function(root) {
 'use strict';
 
+/**
+ * Begin Marked
+ *
+ * TODO: None of our tests actually test a user just passing src?
+ *       `console.log(marked.receivedOnlySource(src, opt, callback));`
+ */
+
+/**
+ * @param  src string
+ * @param  opt (optional) Object
+ * @param  callback (optional) Object
+ *
+ * @return string
+ */
+function marked(src, opt, callback) {
+  // ^^ Can't we do something like marked(src, opt = {}, callback = function())
```

Comment:
Default parameters are ES6 [according to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#Specifications).  Maybe in 2.0.

Reply:
ES5 is gonna probably start to annoy me soon. :)

Reply:
Perhaps we can find a transpiler for 1.0.